db/migrate/20221031194243_create_movies.rb

```diff
@@ -0,0 +1,13 @@
+class CreateMovies < ActiveRecord::Migration[5.2]
+  def change
+    create_table :movies do |t|
+      t.string :movie_title, null: false
+      t.string :movie_director
+      t.string :movie_image
+      t.string :movie_year, null: false
+      t.string :movie_description
```

Comment:
these column names are going to be very repetitive when you have to call `movie.movie_title` as opposed to `movie.title`. where possible I suggest _not_ duplicating table name in the column names